CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+Thanks for wanting to contribute to Diesel! We are very much looking forward to your suggestions, bug reports, and pull requests.
+
+## Setting up diesel locally
+
+1. Install Rust using [rustup], which allows you to easily switch between Rust versions.
+2. Install the system libraries needed to interface with the database systems you which to use.
+
+    These are the same as when compiling diesel. In general, it is a good idea to have _all_ drivers installed so you can run all tests locally.
+
+    *Shortcut:* On macOS, you don't need to install anything to work with Postgres and SQLite.
```

Comment:
macOS ships with libpq, but it does not ship with a postgres database. Most people will need to install postgres to get it running.